io/net/uperf_test.py

```diff
@@ -198,7 +200,7 @@ def tearDown(self):
         """
         Killing Uperf process in peer machine
         """
-        if self.networkinterface:
+        if self.networkinterface and self.uperf_run == '1':
```

Comment:
@vaishnavibhat This condition always fails based on your new variable. Since `self.uperf_run` not changing after assigned with 0. are we trying to avoid this block of code from executions?

Reply:
> @vaishnavibhat This condition always fails based on your new variable. Since `self.uperf_run` not changing after assigned with 0. are we trying to avoid this block of code from executions?

self.uperf_run is set to 1 when UPERF_SERVER_RUN =1 . This happens when the uperf server is running.  The patch takes care of stopping the uperf server only if the UPERF_SERVER_RUN =1 / self.uperf_run is set to 1. 

I might have to take care of the cleanup part in the peer  and need to check if the below would be a better approach. 
```
if self.networkinterface:
        self.uperf_run == '1':
            self.obj.stop()
        cmd = "pkill uperf; rm -rf /tmp/uperf-master"
        output = self.session.cmd(cmd)
```

Reply:
ya this looks better to me

Reply:
Instead of using values 0 & 1, boolean variable can be used with False/True.

Reply:
@vaishnavibhat Please follow the True or False  .. and reflect same in netperf and iperf...  the reason being... checking any condition if it is enabled/disabled or running/notrunning ... always good to have boolean...

I see we are passing 1 or 0  as integer in yaml... in code we are checking as string self.uperf_run == '1'   single quote ..so to avoid this better to go for boolean

you can send the changes to other test too